modules/claims_api/app/swagger/claims_api/v1/form_0966_controller_swagger.rb

```diff
@@ -129,8 +130,9 @@ class Form0966ControllerSwagger
                 property :data do
                   key :type, :object
                   key :required, [:attributes]
-                  key :example, type: 'form/0966', attributes: { type: 'compensation' }
+                  key :example, JSON.parse(File.read(EXAMPLE_PATH))
                   property :attributes do
+                    key :required, %i[participant_claimant_id participant_vet_id signature_indicated]
```

Comment:
I don't think these should be required, only optional and we should be pulling participant_vet_id from MVI when not included.

Reply:
this was been fixed